client/src/sass/components/_sidebar-filter.scss

```diff
@@ -42,7 +42,7 @@
       }
     }
 
-    &--heading {
+    &--expando {
```

Comment:
Here, `sidebar-filter__item--heading` is basically removed.

If this is expected, the class should be removed from the `className` of components.

Reply:
Good call, missed a bit on that cleanup. Restored --heading for consistency with other styles, and it was still used on disk usage.